vocabularies/report-status.ttl

```diff
@@ -0,0 +1,48 @@
+@prefix dc: <http://purl.org/dc/elements/1.1/> .
+@prefix dct: <http://purl.org/dc/terms/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix rptst: <http://linked.data.gov.au/def/report-status/> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+<http://linked.data.gov.au/def/report-status> a owl:Ontology .
+
+rptst:conceptScheme a skos:ConceptScheme ;
+    dct:created "2020-03-03T13:30:19+10:00"^^xsd:dateTime ;
+    dct:creator <http://linked.data.gov.au/org/gsq> ;
+    dct:modified "2020-03-03T13:30:19+10:00"^^xsd:dateTime ;
+    dct:source "Compiled by the Geological Survey of Queensland" ;
+    skos:definition "The status of a report during the submission process in the GSQ Open Data Portal Lodegement System"@en ;
+    skos:hasTopConcept rptst:in-progress,
+        rptst:quarantined,
+        rptst:submitted ;
+    skos:prefLabel "Report Status"@en .
+
+<http://linked.data.gov.au/org/gsq>
+    a sdo:Organization ;
+    sdo:name "Geological Survey of Queensland" ;
+.
+
+rptst:in-progress a skos:Concept ;
+    skos:altLabel "Pending"@en,
+        "Saved"@en,
+        "Started"@en ;
+    skos:definition "The report is in-progress and saved in the lodgement portal awaiting submission."@en ;
+    skos:inScheme rptst:conceptScheme ;
+    skos:prefLabel "In-Progress"@en ;
+    skos:topConceptOf rptst:conceptScheme .
+
+rptst:quarantined a skos:Concept ;
+    skos:definition "The report is held by the department for examination prior to submission being allowed"@en ;
+    skos:inScheme rptst:conceptScheme ;
+    skos:prefLabel "Quarantined"@en ;
+    skos:topConceptOf rptst:conceptScheme .

```

Comment:
rptst:screening a skos:Concept ;
    skos:definition "The report is held by the department for examination prior to submission being allowed"@en ;
    skos:inScheme rptst:conceptScheme ;
    skos:prefLabel "Screening"@en ;
    skos:topConceptOf rptst:conceptScheme .